### DISCUSSION TITLE
Permissions do not work

### DISCUSSION THREAD

**Discussion by oscar-int** [AppIntro/AppIntro#1173] [Q&A] [answered]
Hello, I am using the library, I already implemented the library for AndroidX, and everything worked fine, until I wanted to ask for a permission as in the example, but this appears as an error, I do not see any permission available, for example there I write CAMERA, but it does not allow it and I try another permission and it is not available either, why does this happen? I apologize if my mistake is a rookie :( 

![Screenshot 2023-12-21 at 1 35 21 AM](https://github.com/AppIntro/AppIntro/assets/18153592/ea03c88f-cdb5-4507-b68f-4a40281e9e8b)

**Comment by cortinico** [comment]
> but this appears as an error

Does your code compile?

**Reply by oscar-int** [reply]
Hi, No, I get the typical red stripe in the class title, and then it does not run :(

**Reply by cortinico** [reply, marked as answer]
Do you have the correct import like this one?
https://github.com/AppIntro/AppIntro/blob/bdec42e902242b98a6376f391a0e3407375cf389/example/src/main/java/com/github/appintro/example/ui/permissions/PermissionsIntro2.kt#L3

**Reply by oscar-int** [reply]
Yes, that was the problem, so simple 🤦🏻‍♂️ haha, thank you very much for your help!
